studio/schemas/documents/supportedLanguages.ts

```diff
@@ -0,0 +1,71 @@
+import { defineType } from "sanity";
+import { languages } from "languages";
+
+const languageOptions = languages.map((language) => {
+  return { title: language.title, value: language.id };
+});
+
+export const supportedLanguagesID = "supportedLanguage";
```

Comment:
nitpicky, but the id can also be plural as the document has an array of languages
```suggestion
export const supportedLanguagesID = "supportedLanguages";
```